src/server/editorServices.ts

```diff
@@ -1350,6 +1353,17 @@ namespace ts.server {
             }
         }
 
+        /*@internal*/
+        onExtendedConfigChangedForConfiguredProject(project: ConfiguredProject, extendedConfigFile: string) {

```

Comment:
The call should only take in extendedConfigFile as path and look in shared map and schedule reload for all the projects.
There is no need to `logExtendedConfigFileWatchUpdate`